languages/de.yaml

```diff
@@ -15,6 +15,13 @@ PLUGIN_GANALYTICS:
   POSITION_HELP: "Position des Codes im HTML-Dokument (Head oder Body)"
   POSITION_HEAD: "Kopf"
   POSITION_BODY: "Inhaltsende"
+  DO_NOT_TRACK_SECTION: "Do Not Track"
+  DO_NOT_TRACK: "Aktiviere 'Do Not Track'"
+  DO_NOT_TRACK_HELP: "Erlaubt Klienten, durch das senden des 'Do Not Track'-Headers, vom Tracking ausgeschlossen zu werden."
```

Comment:
Please use this translation `Erlaubt Nutzern, durch das Senden des 'Do Not Track'-Headers, vom Tracking ausgeschlossen zu werden.`